tests/Unit/PointwiseFunctions/EquationsOfState/TestHelpers.hpp

```diff
@@ -178,6 +174,23 @@ void check(EosType in_eos, const std::string& python_function_prefix,
                      std::make_index_sequence<EosType::thermodynamic_dim - 1>{},
                      member_args...);
 }
+template <class Eos, class DataType>
+void test_circular_computation_1d(const std::unique_ptr<Eos>& eos,
+                                  const DataType& used_for_size) {
+  std::random_device r;
+  const auto seed = r();
+  CAPTURE(seed);
+  std::mt19937 generator(seed);
+  std::uniform_real_distribution<> distribution(1.0, 1.0e+04);
+  const auto nn_generator = make_not_null(&generator);
+  const auto nn_distribution = make_not_null(&distribution);
+  const auto specific_enthalpy = make_with_random_values<Scalar<DataType>>(
```

Comment:
I didn't add anything else because in the file for the polytropic eos (PolytropicFluid.cpp) the only two quantities which can be circularly computed are density and enthalpy. Every other thermo quantity is obtained only from density (eg there is a pressure_from_density function but no density_from_pressure).